code/modules/plumbing/plumbers/synthesizer.dm

```diff
@@ -83,9 +83,10 @@
 		var/datum/reagent/R = GLOB.chemical_reagents_list[A]
 		if(R)
 			var/chemname = R.name
-			if(is_hallucinating && prob(5))
-				chemname = "[pick_list_replacements("hallucination.json", "chemicals")]"
-			chemicals.Add(list(list("title" = chemname, "id" = ckey(R.name))))
+			if(HAS_TRAIT(user, TRAIT_CHEMISTRY))
+				if(is_hallucinating && prob(5))
+					chemname = "[pick_list_replacements("hallucination.json", "chemicals")]"
+				chemicals.Add(list(list("title" = chemname, "id" = ckey(R.name))))
```

Comment:
Since you changed how the chemistry trait works, is this even needed?

Reply:
Hi thanks for the review.

I didn't fucking clean it up like I thought I did, fuck

Reply:
Latest Commit Resolved